src/main/java/org/cascadebot/cascadebot/commands/management/permission/GroupPermissionMoveSubCommand.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.commands.management.permission;
+
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import net.dv8tion.jda.core.entities.Member;
+import org.cascadebot.cascadebot.UnicodeConstants;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandExecutable;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.data.objects.GuildPermissions;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import org.cascadebot.cascadebot.permissions.objects.Group;
+import org.cascadebot.cascadebot.utils.PermissionCommandUtils;
+import org.cascadebot.cascadebot.utils.buttons.Button;
+import org.cascadebot.cascadebot.utils.buttons.ButtonGroup;
+
+public class GroupPermissionMoveSubCommand implements ICommandExecutable {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        if (context.getData().getPermissions().getMode() == GuildPermissions.PermissionMode.MOST_RESTRICTIVE) {
+            context.getTypedMessaging().replyDanger("Cannot move groups in most restrictive mode");
```

Comment:
Could you add a todo here to provide a link to the docs for a more detailed explanation?